extra-games/polymc/autobuild/build

```diff
@@ -0,0 +1,11 @@
+abinfo "Building polymc ..."
+mkdir -pv "$BLDDIR"
+cd "$BLDDIR"
+cmake "$SRCDIR" -DCMAKE_BUILD_TYPE=Release \
```

Comment:
Nit: Include `${CMAKE_DEF}` when invoking cmake from custom build scripts.

Reply:
They define `CMAKE_BUILD_TYPE` and `CMAKE_INSTALL_PREFIX` so you don't have to

Reply:
> Nit: Include `${CMAKE_DEF}` when invoking cmake from custom build scripts.

Yeah, and `Release` is wrong, you need `RelWithDebInfo`. But you should use CMAKE_DEF, as @cthbleachbit suggests.